bindings/python/iota_sdk/types/transaction_metadata.py

```diff
@@ -25,96 +25,98 @@ class TransactionFailureReason(Enum):
+    BlockIssuerNotExpired = 34,
+    BlockIssuerExpiryTooEarly = 35,
+    ManaMovedOffBlockIssuerAccount = 36,
+    AccountLocked = 37,
+    TimelockCommitmentInputMissing = 38,
+    TimelockNotExpired = 39,
+    ExpirationCommitmentInputMissing = 40,
+    ExpirationNotUnlockable = 41,
+    ReturnAmountNotFulFilled = 42,
+    NewChainOutputHasNonZeroedId = 43,
+    ChainOutputImmutableFeaturesChanged = 44,
+    ImplicitAccountDestructionDisallowed = 45,
+    MultipleImplicitAccountCreationAddresses = 46,
+    AccountInvalidFoundryCounter = 47,
+    AnchorInvalidStateTransition = 48,
+    AnchorInvalidGovernanceTransition = 49,
+    FoundryTransitionWithoutAccount = 50,
+    FoundrySerialInvalid = 51,
+    DelegationCommitmentInputMissing = 52,
+    DelegationRewardInputMissing = 53,
+    DelegationRewardsClaimingInvalid = 54,
+    DelegationOutputTransitionedTwice = 55,
+    DelegationModified = 56,
+    DelegationStartEpochInvalid = 57,
+    DelegationAmountMismatch = 58,
+    DelegationEndEpochNotZero = 59,
+    DelegationEndEpochInvalid = 60,
+    CapabilitiesNativeTokenBurningNotAllowed = 61,
+    CapabilitiesManaBurningNotAllowed = 62,
+    CapabilitiesAccountDestructionNotAllowed = 63,
+    CapabilitiesAnchorDestructionNotAllowed = 64,
+    CapabilitiesFoundryDestructionNotAllowed = 65,
+    CapabilitiesNftDestructionNotAllowed = 66,
+    SemanticValidationFailed = 255,
 
     def __str__(self):
         return {
             0: "Null.",
-            1: "Transaction type is invalid.",
-            2: "Transaction is conflicting.",
-            3: "Input already spent.",
-            4: "Input creation slot after tx creation slot.",
-            5: "Signature in unlock is invalid.",
-            6: "Commitment input required with reward or BIC input.",
-            7: "Commitment input references an invalid or non-existent commitment.",
-            8: "BIC input reference cannot be loaded.",
-            9: "Reward input does not reference a staking account or a delegation output.",
-            10: "Staking rewards could not be calculated due to storage issues or overflow.",
-            11: "Delegation rewards could not be calculated due to storage issues or overflow.",
-            12: "Inputs and outputs do not spend/deposit the same amount of base tokens.",
-            13: "Under- or overflow in Mana calculations.",
-            14: "Inputs and outputs do not contain the same amount of Mana.",
-            15: "Mana decay creation slot/epoch index exceeds target slot/epoch index.",
-            16: "Native token amount must be greater than zero.",
-            17: "Native token sum exceeds max value of a uint256.",
-            18: "Native token sums are unbalanced.",
-            19: "Multi address length and multi unlock length do not match.",
-            20: "Multi address unlock threshold not reached.",
-            21: "Multi unlocks can't be nested.",
+            1: "Transaction is conflicting.",
```

Comment:
In Rust this is `transaction was conflicting and was rejected.`

Reply:
Good catch, changed